Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+"e1839a8" = {path = ".", editable = true}
```

Comment:
This line basically means that pipenv will run `python setup.py install`, which will pull the install dependencies from `setup.py` according to the Python environment.

Reply:
Any idea what "e1839a8" is?

Reply:
It's a short hash of the path (in this case `.`) automatically generated by pipenv to avoid collisions if you have multiple entries. I think it can actually be anything unique -- I'll try renaming it to `stripe` for clarity.